### PR TITLE
Update edge-view to work with the latest pillar 

### PR DESCRIPTION
**Pull request by milan-zededa**
After introducing support for Local NI with multiple ports, few fields in the `NetworkInstanceStatus` and `NetworkInstanceMetrics` have changed.
Also, `VolumeRefConfig` no longer has the `RefCount` field.
Edge-view needs to be updated to reflect all these changes.

### CODE REVIEW

**Review by christoph-zededa — APPROVED** [review]

**Comment by milan-zededa** [comment]
It seems that Yetus/golangcilint does not support Go 1.22
However, we are already using this Go version for pillar and I would like to keep it unified across packages.

**Comment by milan-zededa** [comment]
I have no idea why Yetus keeps failing. The output has much less files than usually.

File: pkg/edgeview/src/network.go [hunk 1]
```diff
@@ -271,18 +274,28 @@ func doAppNet(status, appstr string, isSummary bool) string {
...
-		if niStatus.RunningUplinkProbing {
-			for _, p := range niMetrics.ProbeMetrics.IntfProbeStats {
-				upStatus := "DOWN"
-				if p.NexthopUP && p.RemoteUP {
-					upStatus = "UP"
+		for _, route := range niStatus.CurrentRoutes {
+			dstNet := route.DstNetwork.String()
+			if route.GatewayApp != emptyUUID {
+				fmt.Printf("   - route dst: %s, gw app: %s\n",
+					route.DstNetwork, route.GatewayApp)
```
Comment (naiming-zededa) [inline pkg/edgeview/src/network.go:281]: can we also add the 'Gateway' in the display of the routes?

**Review by naiming-zededa — COMMENTED** [review]
LGTM

**Review by eriknordmark — APPROVED** [review]
LGTM

File: pkg/edgeview/src/network.go (hunk 1 above)
Comment (milan-zededa) [reply pkg/edgeview/src/network.go]: Added

**Pull request merged**